core/src/components/modal/test/basic/modal.e2e.ts

```diff
@@ -0,0 +1,174 @@
+import { expect } from '@playwright/test';
+import { test } from '@utils/test/playwright';
+
+test.describe('modal: focus trapping', () => {
```

Comment:
Now that we're manually opening/checking all modals, it looks like `openModal` and `testModal` in `test.utils.ts` are no longer needed. Could we remove that file as well?